### PR TITLE
Copyedit readme following Fivetran docs style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Part of https://github.com/fivetran/engineering/issues/80441

Fix grammar and spelling problems. Replace deprecated name "Bing Ads" with "Microsoft Advertising."

I've left a question and additional explanations about my edits inline.

**We need to replace "Bing Ads" everywhere else too.** Kristin confirmed that she will work on this via Slack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 The main focus of the package is to transform the core ad object tables into analytics-ready models, including an 'ad adapter' model that can be easily unioned in to other ad platform packages to get a single-view.
 
 ## Models
 
-This package contains transformation models, designed to work simultaneously with our [Bing Ads source package](https://github.com/fivetran/dbt_bing_ads_source). A depenedency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
+This package contains transformation models that are designed to work simultaneously with our [Microsoft Advertising source package](https://github.com/fivetran/dbt_microsoft_ads_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
 
 | **model**                 | **description**                                                                                                  |
```
Comment (fivetran-chloe) [inline README.md:11]: @fivetran-kristin Are the model names going to change once we update this package's name?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# Bing Ads 
+# Microsoft Advertising
```
Comment (fivetran-chloe) [inline README.md:1]: I know the name of the repo will be `dbt_microsoft_ads` for brevity's sake, but I think we should use the correct name whenever possible.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 The main focus of the package is to transform the core ad object tables into analytics-ready models, including an 'ad adapter' model that can be easily unioned in to other ad platform packages to get a single-view.
 
 ## Models
 
-This package contains transformation models, designed to work simultaneously with our [Bing Ads source package](https://github.com/fivetran/dbt_bing_ads_source). A depenedency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
+This package contains transformation models that are designed to work simultaneously with our [Microsoft Advertising source package](https://github.com/fivetran/dbt_microsoft_ads_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
```
Comment (fivetran-chloe) [inline README.md:9]: I updated the modeling package link to reflect the new repo name that Kristin's implementing.

File: README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ vars:
     bing_ads_database: your_schema_name
 ```
 
-For additional configurations for the source models, please visit the [Bing Ads source package](https://github.com/fivetran/dbt_bing_ads_source).
+For additional configurations for the source models, visit the [Microsoft Advertising source package](https://github.com/fivetran/dbt_microsoft_ads_source).
```
Comment (fivetran-chloe) [inline README.md:35]: I updated the source package link to reflect the new repo name that Kristin's implementing.

File: README.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ This package contains transformation models, designed to work simultaneously wit
 Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 
 ## Configuration
-By default this package will look for your Bing Ads data in the `bing_ads` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Bing Ads data is, please add the following configuration to your `dbt_project.yml` file:
+By default, this package looks for your Microsoft Advertising data in the `bing_ads` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Microsoft Advertising data is, add the following configuration to your `dbt_project.yml` file:
```
Comment (fivetran-chloe) [inline README.md:22]: Kristin confirmed via Slack that `bing_ads` is still the correct schema name.

**Review by fivetran-kristin — APPROVED** [review]

**Pull request merged**